lib/engine/Cargo.toml

```diff
@@ -11,9 +11,9 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-wasmer-types = { path = "../wasmer-types", version = "1.0.0-alpha.1" }
-wasmer-compiler = { path = "../compiler", version = "1.0.0-alpha.1", default-features = false }
```

Comment:
These default features were just `serde` and `std`? Nothing about particular compilers or engines?

Reply:
Yeah, that's correct